ios-trivia-game/ViewControllers/GameOptionsViewController.swift

```diff
@@ -10,17 +10,61 @@ import UIKit
 
 class GameOptionsViewController: UIViewController {
 
+    @IBOutlet weak var categoryPicker: UIPickerView!
+    @IBOutlet weak var numOfQuestionsPicker: UIPickerView!
+    
+    var categoryPickerData: [String] = [String]()
+    var numOfQuestionsPickerData: [String] = [String]()
+    
+    var numOfQuestions: Int?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        getCategories()
+    }
+    
+    func getCategories() {
+        JServiceClient.instance.categories(count: 10, offset: 10, success: {(result: Any?) in
+            let triviaCategories = self.convertWithArray(dictionaries: result as! [NSDictionary])
+            for category in triviaCategories {
+                self.categoryPickerData.append(category.title!)
+            }
+            self.setupPickerData()
+            }, failure: { (error: Error?) in
+                    print("Error: \(error?.localizedDescription)")
+            })
+    }
+    
+    func convertWithArray(dictionaries: [NSDictionary]) -> [TriviaCategory] {
+        var categories = [TriviaCategory]()
+        
+        for dictionary in dictionaries {
+            let category = TriviaCategory(dictionary: dictionary)
+            categories.append(category)
+        }
+        
+        return categories
+    }
+    
+    func setupPickerData() {
+        categoryPicker.delegate = self
+        categoryPicker.dataSource = self
+        numOfQuestionsPicker.delegate = self
+        numOfQuestionsPicker.dataSource = self
+        self.numOfQuestionsPickerData = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10"]
```

Comment:
We can alternatively do something like: 

```
array = Array(0...10) 
```

[Link](http://stackoverflow.com/questions/27685277/is-there-a-way-to-instantly-generate-an-array-filled-with-a-range-of-values-in-s)
However, I'm not sure if it needs to be an array of strings. If it does, then the hard-coded array would be fine.